### PR TITLE
Eng 46 learner dashboard layout

### PR DESCRIPTION
**Pull request by chrisbendel**

### CODE REVIEW

File: frontend/src/screens/learner/card.tsx [hunk 1]
```diff
@@ -36,6 +36,19 @@ const Card = styled(Box)({
         minHeight: 200,
         maxHeight: 200,
     },
+    [media.tablet]: {
```
Comment (chrisbendel) [inline frontend/src/screens/learner/card.tsx:39]: To avoid cards overlapping on tablet screens

File: frontend/src/screens/learner/learner-welcome-modal.tsx [hunk 2]
```diff
@@ -2,18 +2,17 @@ import React, { useState } from 'react'
...
-}> = ({ demographicSurvey }) => {
+export const LearnerWelcomeModal: FC = () => {
     const [open, setOpen] = useState(true)
     const api = useApi()
-    const { data: preferences, refetch } = useUserPreferences()
+    const { data: preferences, refetch, isLoading } = useUserPreferences()
+    const { demographicSurvey } = useParticipantStudies();
 
-    if (!demographicSurvey || demographicSurvey.completedAt || preferences?.hasViewedWelcomeMessage) {
+    if (isLoading || !demographicSurvey || demographicSurvey.completedAt || preferences?.hasViewedWelcomeMessage) {
```
Comment (chrisbendel) [inline frontend/src/screens/learner/learner-welcome-modal.tsx:15]: Preventing a quick flashing when returning to studies dashboard from the landing page

File: backend/app/services/launch_pad.rb [hunk 3]
```diff
@@ -33,19 +33,16 @@ def launch
   def land(consent: true, aborted: nil)
     if aborted
       self.abort(aborted)
-      return { aborted_at: Time.now }
```
Comment (chrisbendel) [inline backend/app/services/launch_pad.rb:36]: These changes are mainly to prevent another unnecessary call to the backend from the landing component namely [here](https://github.com/openstax/kinetic/pull/349/files#diff-44e68b70a9d2c00bafd45edf2cb54150beaeee6f98225cdb4c23ba3e7c5fe71fL28)

We can just return the updated launched study which should have all of the appropriate completed/aborted information

File: backend/spec/requests/api/v1/participant/studies_spec.rb [hunk 4]
```diff
@@ -207,7 +207,6 @@
 
       it 'returns completion status for a multistage study' do
         stage1a.launch_by_user!(user1)
-        # expect_any_instance_of(LaunchPad).to receive(:land)
```
Comment (chrisbendel) [inline backend/spec/requests/api/v1/participant/studies_spec.rb:210]: We can use the actual LaunchedStudy data coming back now

File: frontend/src/screens/learner/studies.ts
Comment (chrisbendel) [inline frontend/src/screens/learner/studies.ts:1]: This ended up being a brutal diff, i'm sorry in advance 😢 

TLDR: Use react query to fetch participant studies, do a similar approach to organize the highlighted, completed, etc

Added a custom hook to handle the search logic for the learner dashboard

File: frontend/src/screens/study-landing.tsx [hunk 5]
```diff
@@ -55,9 +45,10 @@ export default function StudyLanding() {
             params['aborted'] = LandStudyAbortedEnum.Refusedconsent
         }
 
-        landStudy(api, params)
-            .then(setLanded)
-            .catch(setError)
+        landStudy.mutate(params, {
```
Comment (chrisbendel) [inline frontend/src/screens/study-landing.tsx:48]: Playing around with the mutation feature, pretty neat
